x-pack/plugins/triggers_actions_ui/public/application/sections/action_connector_form/action_type_form.tsx

```diff
@@ -115,7 +115,7 @@ export const ActionTypeForm = ({
       }
     }
     // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [actionItem.group, defaultParams]);
```

Comment:
This is the important part of the fix.
It ensures that we only reapply the `defaultParams` when the group has been changed.
Ideally we'd only do so when the group has changed *and* the params aren't the same as what's already there, but this doesn't work because it causes it to overwrite non-default values selected by the user.

In the future we should apply the default to the actual action object in memory a level above, instead of the ActionForm doing it. But that's a bigger fix that would cause Main to remain broken for longer.